docker-compose.yml

```diff
@@ -1,4 +1,5 @@
 version: '3.5'
+
```

Comment:
Extra line not needed

Reply:
Although not needed, think it reads better with the version being separated from the services. What do you think?

Reply:
Tbh, you're right 😄